sentinel-adapter/sentinel-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/DubboAppContextFilter.java

```diff
@@ -36,7 +36,7 @@
     public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcException {
         String application = invoker.getUrl().getParameter(Constants.APPLICATION_KEY);
         if (application != null) {
-            RpcContext.getContext().setAttachment(DubboUtils.DUBBO_APPLICATION_KEY, application);
+            RpcContext.getContext().setAttachment(DubboUtils.SENTINEL_APPLICATION_KEY, application);
```

Comment:
Using `SENTINEL_APPLICATION_KEY` may confuse users as the attachment value refers to application name in Dubbo, not Sentinel appName.

Reply:
the key inserted into dubbo context, so marked as sentinel key may be much correcter than dubbo key.

Reply:
@sczyh30

Reply:
@CarpenterLee What do you think of here?

Reply:
I prefer @kimmking 's opinion. Since we use `DubboAppContextFilter` to modify the Dubbo's context, uses should know that the attachment belongs to Sentinel, instead of Dubbo.

Reply:
That's OK, but we need to add a comment on this field to indicate that its value refers to application name configured in Dubbo, not Sentinel appName (though our document has emphasized that).